src/Gui/DlgSettingsColorGradientImp.cpp

```diff
@@ -54,7 +55,10 @@ DlgSettingsColorGradientImp::DlgSettingsColorGradientImp( QWidget* parent, Qt::W
     ui->floatLineEditMin->setValidator(fMinVal);
 
     QFontMetrics fm(ui->floatLineEditMax->font());
-    ui->floatLineEditMax->setMinimumWidth(QtTools::horizontalAdvance(fm, QString::fromLatin1("-1000.000000")));
+    ui->floatLineEditMax->setMinimumWidth(QtTools::horizontalAdvance(fm, QString::fromLatin1("-1000.0")));

```

Comment:
It is on purpose to use a number with six decimals after the dot to guarantee that the dialog is wide enough. With one of your recent changes you have changed the width in the UI file and every time I opened the dialog I had to manually change its size in order to avoid that numbers are truncated.

Reverting this fix again will be an annoying usability regression.

Reply:
> It is on purpose to use a number with six decimals after the dot to guarantee that the dialog is wide enough.

But this is a hack. A minimum for the width in the UI is the right approach, Alternatively one can set in the UI to expand if the numbers become bigger. I will have at the UI file and make this change.

> Reverting this fix again will be an annoying usability regression.

I obviously missed something. What do you refer to? Did I accidentally remove the minimal width settings from this dialog? Or what do you mean with the fix?

Reply:
> I will have at the UI file and make this change.

This is the result: #6695 I found it annoying that one had to deal with huge numbers because the LineEdit did not handle scientific notation. But most simulation results need this. Can you please give it a try if this change if fine for you as well?